frontend/javascripts/viewer/view/context_menu.tsx

```diff
@@ -619,6 +620,15 @@ function getNodeContextMenuOptions({
       onClick: () => Store.dispatch(setActiveNodeAction(clickedNodeId)),
       label: "Select this Node",
     },
+    activeTreeId === clickedTree.treeId
```

Comment:
wouldn't it be useful if the user could also focus the tree even when it was not active before? if this is only possible by activating the tree first, I'd be fine with that.

Reply:
they can by simply selecting the node from the context menu. I felt like having two context menu actions doing the exact same thing but with a different label was unnecessary. I could change the label of the `Select this Node` to `Select Node and Focus Tree` if the clicked node is not from the active tree maybe, to make this more clear?

Reply:
> they can by simply selecting the node from the context menu.

ah, good point. then, let's leave it as is :)